Cargo.toml

```diff
@@ -156,10 +156,10 @@ tezos-smart-rollup-entrypoint = { git = "https://gitlab.com/tezos/tezos.git" }
 tezos-smart-rollup-debug = { git = "https://gitlab.com/tezos/tezos.git" }
 tezos-smart-rollup-panic-hook = { git = "https://gitlab.com/tezos/tezos.git" }
 tezos-smart-rollup-storage = { git = "https://gitlab.com/tezos/tezos.git" }
-boa_ast = { git = "https://github.com/trilitech/boa.git", rev = "78c8540e" }
-boa_engine = { git = "https://github.com/trilitech/boa.git", rev = "78c8540e" }
-boa_gc = { git = "https://github.com/trilitech/boa.git", rev = "78c8540e" }
-boa_interner = { git = "https://github.com/trilitech/boa.git", rev = "78c8540e" }
-boa_macros = { git = "https://github.com/trilitech/boa.git", rev = "78c8540e" }
-boa_parser = { git = "https://github.com/trilitech/boa.git", rev = "78c8540e" }
-boa_profiler = { git = "https://github.com/trilitech/boa.git", rev = "78c8540e" }
+boa_ast = { git = "https://github.com/trilitech/boa.git", branch = "ryutamago/fix-undefined-to-json" }
+boa_engine = { git = "https://github.com/trilitech/boa.git", branch = "ryutamago/fix-undefined-to-json" }
+boa_gc = { git = "https://github.com/trilitech/boa.git", branch = "ryutamago/fix-undefined-to-json" }
+boa_interner = { git = "https://github.com/trilitech/boa.git", branch = "ryutamago/fix-undefined-to-json" }
+boa_macros = { git = "https://github.com/trilitech/boa.git", branch = "ryutamago/fix-undefined-to-json" }
+boa_parser = { git = "https://github.com/trilitech/boa.git", branch = "ryutamago/fix-undefined-to-json" }
+boa_profiler = { git = "https://github.com/trilitech/boa.git", branch = "ryutamago/fix-undefined-to-json" }
```

Comment:
🔎 🤔

Reply:
Changed since `main` had to point to an old rev for the patches older version for #594. The patches newer version is at `"ryutamago/fix-undefined-to-json"`

Reply:
Oh I see